### ISSUE TITLE
tag 0.2.0 request/request-deferred for melpa-stable?

### ISSUE THREAD

**Issue by ardumont** [tkf/emacs-request#18]
Hello,

context: 
org-trello uses request-deferred on melpa (ok) and melpa-stable (ko).
org-trello cannot be installed correctly on melpa-stable because only 0.1.0 version exists on it.
This diverges from the last version according to https://github.com/tkf/emacs-request/commit/8b61b71ff5faeeea88f4560ed9230299500bab0c.

I do not know if you know this but the melpa team has come up with an equivalent melpa-stable.
Regarding melpa-stable's github build, this means it uses git tag to determine stable version.

As you only tag it as 0.1.0,  it appears only as 0.1.0 here - http://melpa-stable.milkbox.net/#/request-deferred and http://melpa-stable.milkbox.net/#/request

I wondered if it could be possible for you to bump a tag 0.2.0 on github please?

Cheers,

*Note* Thanks for request, it's pretty neat.

**Comment by ardumont** [comment]
I wish :D

**Comment by rodprice** [comment]
Installation of EIN from melpa-stable fails because EIN depends on request v. 0.2.0, while melpa-stable only has request 0.1.0. Your source files say the version is 0.2.0. Would you take a moment, please, and create a v0.2.0 tag? Once that is done, melpa-stable will vacuum it up, and people will be able to install EIN from melpa-stable. Thanks.

**Comment by ardumont** [comment]
+1

**Comment by markokocic** [comment]
+1

btw, is thes package still maintained? Last commit on master is more than 1.5 years old.

**Comment by ardumont** [comment]
Hello,

I discovered in an org-trello issue that @abingham took over.
And indeed, the [melpa recipe now targets his fork](https://github.com/milkypostman/melpa/blob/master/recipes/request)
I am unfortunately unable to post issues on his fork.

@abingham, are you able to tag from your fork? If so, can you please do the 0.2.0 tag?

Thanks anyway.

Cheers,

**Comment by abingham** [comment]
I should be able to add tags. Also, I've added support for issues on my fork; thanks for pointing that out!

**Comment by ardumont** [comment]
Great double news!

Thanks.

**Comment by abingham** [comment]
I've created a new v0.2.0 tag now.

**Comment by ardumont** [comment]
Cool, now we can close!

Indeed, [melpa-stable already references the 0.2.0](http://stable.melpa.org/#/request).

Thanks again,

Cheers
